src/http_rest_api_handler.cpp

```diff
@@ -367,13 +374,16 @@ Status HttpRestApiHandler::processModelControlApiRequest(std::string& response)
         return status;
     }
 
-    std::string jsonString;
-    status = GetModelStatusImpl::serializeModelsStatuses2Json(modelsStatuses, jsonString);
+    std::string modelsStatusesJson;
+    status = GetModelStatusImpl::serializeModelsStatuses2Json(modelsStatuses, modelsStatusesJson);
     if (!status.ok()) {
         response = createErrorJsonWithMessage("Serializing model statuses to json failed.");
         return status;
     }
-    response = jsonString;
+
+    std::string pipelinesStatusesJson = manager.getPipelinesStatusesAsJson();
+
+    response = "\"models\" : \n" + modelsStatusesJson + ",\n\"pipelines\" : \n" + pipelinesStatusesJson;
```

Comment:
@dkalinowski I Think we don't use externally(outside OVMS logs) so far "pipeline" name. I am not sure what we would want to report here alongside models.

Pipelines are consistent with OVMS logs, however the feature is called DAG Scheduler.

Reply:
Actually we have pipelines_configs in our config.json so "pipelines" should be OK.

Reply:
We agreed to leave everything under models for now.